Framework/PythonInterface/plugins/algorithms/Abins.py

```diff
@@ -50,17 +50,17 @@ def summary(self):
     def PyInit(self):
 
         # Declare all properties
-        self.declareProperty(name="DFTprogram",
+        self.declareProperty(name="AbInitioProgram",
                              direction=Direction.Input,
                              defaultValue="CASTEP",
                              validator=StringListValidator(["CASTEP", "CRYSTAL", "DMOL3", "GAUSSIAN"]),
-                             doc="DFT program which was used for a phonon calculation.")
+                             doc="An ab initio program which was used for vibrational or phonon calculation.")
 
-        self.declareProperty(FileProperty("PhononFile", "",
+        self.declareProperty(FileProperty("VibrationalOrPhononFile", "",
```

Comment:
Would something like `SimulationFile` be more general for the future?

Reply:
@martyngigg I asked Sanghamitra about those names (I cc. you).